build.gradle

```diff
@@ -1,10 +1,13 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
-    ext.kotlin_version = '1.1.3-2'
-    ext.retrofit_version = '2.1.0'
+    ext.fresco_version = '1.5.0'
+    ext.jackson_version = '2.9.0'
+    ext.kotlin_version = '1.1.4-3'
+    ext.leak_canary_version = '1.5.2'
+    ext.okhttp_version = '3.8.1'
+    ext.retrofit_version = '2.3.0'
     ext.support_lib_version = '26.0.1'
```

Comment:
I've been feeling like it could be a good idea to move app dependency versions to `/app/build.gradle`. That can be work for another day though.